aws/logs_monitoring/lambda_function.py

```diff
@@ -80,6 +80,27 @@ def __init__(self, name, pattern, placeholder):
     )
 ]
 
+
+# Filtering logs 
+# Option to include or exclude logs based on a pattern match
+INCLUDE_AT_MATCH = os.getenv("INCLUDE_AT_MATCH", default=None)
+if INCLUDE_AT_MATCH is not None:
```

Comment:
We can see code duplication here, the exact same logic is also used inside the scrubber, I think we should isolate this logic inside a method, and use from different places.
The interface would look like:
```
def compileRegex(pattern):
  # TODO: implement me
  return None
```

Reply:
I can help with this one if you want, feel free to ping me IRC.

Reply:
I added a `compileRegex` and used it for the include, exclude, and scrubbing rules.